src/librustc_lint/builtin.rs

```diff
@@ -1784,3 +1785,70 @@ impl<'a, 'tcx> LateLintPass<'a, 'tcx> for UnnameableTestFunctions {
         };
     }
 }
+
+declare_lint! {
+    pub ASYNC_IDENTS,
+    Deny,
```

Comment:
Just to confirm, but the `Deny` here was for tests? 

For the other 2018 lints at least they're all set to `Allow` by default and they're only active with the lint group turned on (which rustfix does automatically)

Reply:
I just followed what another 2018 incompat lint did, and that was deny

Reply:
E.g. https://github.com/rust-lang/rust/blob/68c39b9fec17846005da9a8e42991422c08c377c/src/librustc/lint/builtin.rs#L302

Reply:
Aha, that's not a 2018 lint I think because that's changing for the 2015 edition, I was thinking of lints like this one -- https://github.com/rust-lang/rust/blob/68c39b9fec17846005da9a8e42991422c08c377c/src/librustc/lint/builtin.rs#L264-L269

Reply:
Then this is wrong: https://github.com/rust-lang/rust/blob/254f8796b729810846e2b97620032ecaf103db33/src/librustc_lint/lib.rs#L213-L216

Reply:
Ah I believe so! Want to file an issue for that?

Reply:
edition lints can be any level they wish to be, that one probably makes more sense as deny, but this one should be allow